cognitedata/reveal
# feat: create scaffolding for reveal-react-components library

#### Type of change

![Feat](https://img.shields.io/badge/Type-Feat-green)  

#### Jira ticket :blue_book:


https://cognitedata.atlassian.net/browse/REV-917

## Description :pencil:


Initial setup and build for the new reveal-react-components library.

## How has this been tested? :mag:



A .tar publish build was built for a sample component and was successfully mounted in Data Explorer and InRobot.

## Test instructions :information_source:



1. Add `"@cognite/reveal-react-components": "link:../react-components"` as a dependency to examples/package.json
2. In App.tsx `import { Button } from '@cognite/reveal-react-components';`
3. Mount a new Button component in the Page Container, `<Button label={'Hello World!'} />`
4. Witness a beautifully designed button appear!


## Checklist :ballot_box_with_check:
